### PR TITLE
fix: #655 keep border on default buttons

### PR DESCRIPTION
**Pull request by francisprovost**

### CODE REVIEW

File: client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescriptions.tsx [hunk 1]
```diff
@@ -211,6 +211,7 @@ const Prescriptions : React.FC<Props> = ({ prescriptions, clinicalImpressions })
                   <Row>
                     <Col>
                       <Button
+                        className="bordelessButton"
```
Comment (YanLabCHU) [inline client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescriptions.tsx:214]: Also, there is a typo in the class name, it should be `borderlessButton`.
I know you're just using it but could you convert it to `borderless-button` ?

**Review by YanLabCHU — COMMENTED** [review]
small change

File: client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescriptions.tsx (hunk 1 above)
Comment (francisprovost) [reply client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescriptions.tsx:214]: Yeah. I'll rename it `button--borderless` to follow a bit more the bem naming

**Comment by francisprovost** [comment]
@YanLabCHU @claudia1296 ping

**Review by YanLabCHU — APPROVED** [review]

**Pull request merged**